content/posts/blog/2021-01-23_MesagerieInstantanee-Nuesteorbadespreaplicatie.md

```diff
@@ -47,7 +47,7 @@ Dacă sunteți interesați să faceți o alegere pentru următorul deceniu, perm
 - [Noțiuni de bază](https://xmpp.org/getting-started/), XMPP Standards Foundation
 - Articol pe blogul XSF: [It’s all about choices and control](https://xmpp.org/2015/01/its-all-about-choices-and-control/) scris in 2015 de către Laura
 - [Modern XMPP](https://docs.modernxmpp.org/), site web care susține dezvoltatorii și noii veniți să înceapă cu XMPP
-- Comentarii din comunitatea XMPP (nu toate în limba engleză):
+- Comentarii din comunitatea XMPP (în limba engleză sau în alte limbi):
```

Comment:
Thanks, but I meant something like: 
engleză --> românesc

Reply:
I understood, but there's no romanian to begin with, so imho it's more honest to adjust expectations.

But if you really-really want that I can make it say that back.

Reply:
I'm confused, in each article I adjusted this to the thext language. Like German text, warning not everything is German etc.
It does not apply here?

Reply:
I just disagree with your take, because there's a chance there's some french/spanish/german there. But not this locale. :)

Reply:
I don't get it really. I just want to tell them that the following is not in romanian.